### PR TITLE
Bump @stytch/stytch-js~2.0.0

### PR DESCRIPTION
**Pull request by nathan-stytch**
also add node_modules to gitignore

### CODE REVIEW

**Review by mary-stytch — APPROVED** [review]
Sorry I missed this. Can also wait until the changes re: user_created are merged.

File: src/Login.js
Comment (mary-stytch) [inline src/Login.js:7]: Can we keep this public token generic?

Comment (nathan-stytch) [reply src/Login.js]: whoops, https://github.com/stytchauth/stytchjs-react-express/pull/2/commits/3668117a06d5d5b8a9d0aed779f5979e802c278d

**Pull request merged**
